packages/paste-core/components/form/test-utils/index.ts

```diff
@@ -0,0 +1,26 @@
+export interface AttributesMap {
+  class?: string;
+  'data-testid'?: string;
+  id?: string;
+  'aria-invalid'?: string;
+  disabled?: boolean;
+  label?: string;
+  'data-attr'?: string;
+  selected?: string;
+  title?: string;
+  spellcheck?: string;
+  hidden?: boolean;
+  draggable?: boolean;
+  accesskey?: boolean;
+  autocomplete?: boolean;
+  form?: string;
+}
+
+export const createAttributeMap = (element: HTMLElement): AttributesMap => {
+  const attributesNodeList: NamedNodeMap = element.attributes;
+
+  return Object.keys(attributesNodeList).reduce((attributesMap, key) => {
```

Comment:
Yeah, this is much better than my version 😉

Reply:
😸